docs/conf/prague/2020/news/cfp-open.rst

```diff
@@ -0,0 +1,46 @@
+:template: {{year}}/generic.html
+
+.. post:: June 22, 2020
+   :tags: prague-2020, website, cfp, tickets
+
+Announcing Prague Call for Proposals
+====================================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_.
+
+Due to the global situation that you're all familiar with we've moved the Prague conference online, and changed the dates to 19-20 October 2020.
+This means that we're only leaving our call for proposals open for one month. We know that it hasn't been an easy year. Nevertheless we'd love to see you on our online stage in October, to share your ideas with our inspiring community of documentarians.
+
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, editor, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We're hoping that folks who would not be able to travel to Prague will be able to join us this time round, both as attendees and as speakers! If you've always wanted to talk about documentation but not been able to travel for whatever reason, now is the time.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`__ on the website.
+The Call for Proposals will be open until **Midnight {{tz}} on {{cfp.ends}}**.
+
+Tickets
+-------
+
+We'll be opening ticket sales very soon, in the meantime, you can take a look at the new prices for the online conference on our `Tickets <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_ page.
+
+
+Grant tickets
+-------------
+
+We'll be opening our grant program very soon.
```

Comment:
I don't think we need a title for this, we should just introduce the grant program in the above paragraph. We can likely copy from the Portland introduction, explaining a bit about why we're doing it.